pkg/asset/installconfig/installconfig.go

```diff
@@ -4,15 +4,14 @@ import (
 	"os"
 
 	"github.com/ghodss/yaml"
-	"github.com/pkg/errors"
```

Comment:
nit: keep the imports as-is.

Reply:
You don't use `gofmt`? my editor runs it at every file save, so I didn't think that would be a problem.
Happy to oblige, what's your convention?

Reply:
aha, extra line disappeared 👍fixing it.